api/server_helpers_test.go

```diff
@@ -186,6 +186,21 @@ func createServerTester(name string) (*serverTester, error) {
 		}
 	}
 
+	// Announce the host.
+	err = st.announceHost()
+	if err != nil {
+		return nil, err
+	}
+
+	// Set an allowance for the renter.
+	allowanceValues := url.Values{}
+	allowanceValues.Set("funds", "10000000000000000000000000000")
```

Comment:
if you want something more readable, you can use a `types.Currency` and its `String` method.